lnrpc/rpc_utils.go

```diff
@@ -234,3 +236,17 @@ func CalculateFeeRate(satPerByte, satPerVByte uint64, targetConf uint32,
 
 	return feeRate, nil
 }
+
+// SerializeAndHexEncodeTx serializes the given wire transaction and returns
+// its hexadecimal representation. If serialization fails, an error is
+// returned.
+func SerializeAndHexEncodeTx(tx wire.MsgTx) (string, error) {
```

Comment:
Okay. Initially, I considered this too. However, I opted to pass the copy because `SerializeAndHexEncodeTx` doesn't mutate the `tx` in place. I'm fine using the pointer to `wire.MsgTx` if it's commonly used in the codebase.